src/js/modules/infragistics.util.js

```diff
@@ -125,6 +125,23 @@
 
 		// The dummy class constructor
 		function Class() {
+			prototype = this;
+			for (name in prop) {
+				if (prop.hasOwnProperty(name)) {
+
+					// Check if we're overwriting an existing function
+					var isFn = prototype[ name ] = doSuper && typeof prop[ name ] === "function" &&
```

Comment:
Do you need to assign `prototype[ name ]` here as well, even though you set it later?